gradle/formatting.gradle

```diff
@@ -57,57 +61,81 @@ import org.opensearch.gradle.BuildPlugin
 
 allprojects {
   plugins.withType(BuildPlugin).whenPluginAdded {
-      project.apply plugin: "com.diffplug.spotless"
+    project.apply plugin: "com.diffplug.spotless"
+    project.apply plugin: "org.openrewrite.rewrite"
 
-      spotless {
-        java {
-          // Normally this isn't necessary, but we have Java sources in
-          // non-standard places
-          target '**/*.java'
+    spotless {
+      java {
+        // Normally this isn't necessary, but we have Java sources in
+        // non-standard places
+        target '**/*.java'
 
-          removeUnusedImports()
```

Comment:
rewrite has its own flaws, as nobody is perfect.
In this regards it found some leftovers.